cfg/schema.sql

```diff
@@ -30,33 +30,29 @@ SET default_with_oids = false;
 -- Name: account_transactions; Type: TABLE; Schema: public; Owner: -
 --
 
-CREATE TABLE IF NOT EXISTS public.account_transactions (
-    account bytea NOT NULL,
-    ledger_seq bigint NOT NULL,
-    transaction_index bigint NOT NULL
+CREATE TABLE public.account_transactions (
+    account           bytea  NOT NULL,
+    ledger_seq        bigint NOT NULL,
+    transaction_index bigint NOT NULL,
+    trans_id          bytea  NOT NULL
 );
 
 
 --
 -- Name: ledgers; Type: TABLE; Schema: public; Owner: -
 --
 
---CREATE TABLE IF NOT EXISTS public.ledgers (
---    ledger_seq integer NOT NULL,
---    ledger_hash bytea NOT NULL,
---    parent_hash bytea NOT NULL,
---    close_time_human timestamptz NOT NULL
---);
-
---
--- Name: transactions; Type: TABLE; Schema: public; Owner: -
---
-
-CREATE TABLE IF NOT EXISTS public.transactions (
```

Comment:
@mtrippled Was the schema for the transactions table removed? I can't seem to find it.

Reply:
Yes, the tx hash is in the account_transactions table. And I'm assuming that tx() doesn't need the table at all because it looks up the nodestore.